### PR TITLE
es: Improved translation on first two sections

### PR DESCRIPTION
**Pull request by aabreuglez**
Hi, I think the first two sections are more readable now.

### CODE REVIEW

File: README-es.md [hunk 1]
```diff
@@ -21,29 +21,29 @@
 
 ![curl -s 'https://raw.githubusercontent.com/jlevy/the-art-of-command-line/master/README.md' | egrep -o '`\w+`' | tr -d '`' | cowsay -W50](cowsay.png)
 
-La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta mejora su flexibilidad y productividad como ingeniero en formas obvia y sutil. Esta es una selección de notas y consejos al usar el terminal que encontramos útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no es larga, pero si puedes usar y recordar todos los puntos aquí mostrados, sabrás un montón.
+La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta habilidad mejora tu flexibilidad y productividad como ingeniero de formas obvias y sutiles. Esta es una selección de notas y consejos para usar el terminal que encontramos útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no es larga, pero si puedes usar y recordar todos los puntos aquí mostrados, sabrás un montón.
```
Comment (ceoaliongroo) [inline README-es.md]: habilidad:  Change the original idea, please, remove it.

File: README-es.md [hunk 2]
```diff
@@ -21,29 +21,29 @@
...
 
 Este trabajo es el resultado de [muchos autores y traductores](AUTHORS.md).
 La mayor parte
 [originalmente](http://www.quora.com/What-are-some-lesser-known-but-useful-Unix-commands)
 [apareció](http://www.quora.com/What-are-the-most-useful-Swiss-army-knife-one-liners-on-Unix)
 en [Quora](http://www.quora.com/What-are-some-time-saving-tips-that-every-Linux-user-should-know),
-pero debido al interés mostrado, parecía valer la pena usar Github, donde personas más talentosas que el autor original pudiendo fácilmente sugerir mejoras. Si ve un error o algo que podría ser mejor, por favor, crea un issue o PR! (Por supuesto primero revisa la sección meta de PRs/issues.)
+pero debido al interés mostrado, parecía valer la pena usar Github, donde personas más talentosas que el autor original pudieran sugergir mejoras fácilmente. Si encuentras un error o algo que podría mejorarse, por favor, ¡crea un issue o PR! (Por supuesto primero revisa la sección meta de PRs/issues.)
```
Comment (ceoaliongroo) [inline README-es.md]: encuentras: It's different from the original verb used, i suggest use "ves"

File: README-es.md [hunk 3]
```diff
@@ -21,29 +21,29 @@
...
-- Esta escrito para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos aplican o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
-- Se enfoca en Bash interactivo, aunque muchos de los consejos se aplican para otros shells y al Bash scripting por lo general.
-- Incluye tanto comandos "estándar" Unix, así como aquellos que requieren la instalación especial de un paquete -- siempre que sea suficientemente importante para ameritar su inclusión.
+- Esta guía es tanto para principiantes como para expertos. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente puede ahorrar tiempo comparado con otras alternativas.
```
Comment (ceoaliongroo) [inline README-es.md]: Let's keep "experimentados", please remove it.

File: README-es.md [hunk 4]
```diff
@@ -21,29 +21,29 @@
...
+- Está escrita para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos sirven o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
```
Comment (ceoaliongroo) [inline README-es.md]: sirven: It's different from the original verb used. 

Maybe you could suggest a different verb, more related with emplear, usar, aplicar.

File: README-es.md [hunk 5]
```diff
@@ -21,29 +21,29 @@
...
+- Se concentra en Bash interactivo, aunque muchos de los consejos sirven para otros shells y en general para el scripting en Bash.
```
Comment (ceoaliongroo) [inline README-es.md]: concentra: please remove it.
sirven: It's different from the original verb used. Look the previous suggestion...  
"Bash scripting" is used without translation., this change not apply

File: README-es.md [hunk 6]
```diff
@@ -21,29 +21,29 @@
...
+- Incluye tanto comandos "estándar" Unix como algunos que requieren la instalación de un paquete especial -- siempre que sea lo suficientemente importante para justificar su inclusión.
```
Comment (ceoaliongroo) [inline README-es.md]: Please change to:  

Incluye tanto comandos "estándar" de Unix así como algunos que requieren la instalación de un paquete especial -- siempre que sea suficientemente importante para ameritar su inclusión.

File: README-es.md [hunk 7]
```diff
@@ -21,29 +21,29 @@
...
 
 Notas:
 
-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
-- Usa [Explainshell](http://explainshell.com/) para obtener detalles de ayuda sobre que comandos, opciones, pipes, etc.
+- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares buscando en Google una vez conoces la idea o el comando. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
```
Comment (ceoaliongroo) [inline README-es.md]: I don't see this is necessary

File: README-es.md [hunk 8]
```diff
@@ -21,29 +21,29 @@
...
+- Usa [Explainshell](http://explainshell.com/) para obtener una ayuda detallada sobre que hacen los comandos, las opcioneslas pipes, etc.
```
Comment (ceoaliongroo) [inline README-es.md]: I don't see this is necessary

**Comment by ceoaliongroo** [comment]
@alu0100601398 Thanks you very much for you collaboration. 

Please review the suggestion and do changes.

File: README-es.md (hunk 4 above)
Comment (alu0100601398) [inline README-es.md]: I will let "aplican" by know. But it sounds really strange to me. I'll keep thinking about it.

File: README-es.md (hunk 3 above)
Comment (alu0100601398) [inline README-es.md]: Sure, I probably changed the meaning with experts. I went back to "experimentados", but it's a bit weird. We use this word mostly related with sex. (No kidding).

File: README-es.md (hunk 2 above)
Comment (alu0100601398) [inline README-es.md]: Changed to "ves".

File: README-es.md (hunk 1 above)
Comment (alu0100601398) [inline README-es.md]: Removed  "habilidad" and changed from "esta" to "ésta" (is pronoum now).

File: README-es.md (hunk 5 above)
Comment (alu0100601398) [inline README-es.md]: Sorry about that , I meant "centra".
I will let "aplican" here too.
Back to Bash scripting.

File: README-es.md (hunk 6 above)
Comment (alu0100601398) [inline README-es.md]: I've never readed before "ameritar" I changed it to "merecer".

File: README-es.md (hunk 8 above)
Comment (alu0100601398) [inline README-es.md]: Changed the paragraph back. And now modified to: "que hacen los comandos, las opciones, las pipes, etc."

File: README-es.md (hunk 7 above)
Comment (alu0100601398) [inline README-es.md]: I changed it back. I moved the comma.

**Comment by alu0100601398** [comment]
@ceoaliongroo Thanks you for reviewed everything!
I did changes, and I commented all of them.

File: README-es.md (hunk 2 above)
Comment (AndresCidoncha) [inline README-es.md]: In spanish, we use "encontrar un error" better than "ver un error".

File: README-es.md (hunk 3 above)
Comment (AndresCidoncha) [inline README-es.md]: I think that "experimentados" is almost not used. We use "Experto" or "Con experiencia".

File: README-es.md (hunk 2 above)
Comment (ceoaliongroo) [inline README-es.md]: @AndresCidoncha you are right, but still is a bad practice using the spanish, there is not a problem to use this verb in the sentence.

__From Real Academy Dictionary:__
http://lema.rae.es/drae/srv/search?val=ver

Also the most importan think keep the original idea and the spirit of the author.

We keep "ves" instead ëncontrar

File: README-es.md (hunk 3 above)
Comment (ceoaliongroo) [inline README-es.md]: Sorry is your perception.

The use "experimentado" is fine, also meaning expert

__From Real Academy Dictionary:__
http://lema.rae.es/drae/srv/search?id=SpNWvLZYBDXX2sftDQrf

File: README-es.md (hunk 5 above)
Comment (ceoaliongroo) [inline README-es.md]: Don't changes the original idea of the sentence, please keep:
`"... al Bash scripting por lo general."`

Comment (ceoaliongroo) [inline README-es.md]: Please remove:
`como algunos que requieren la instalación de un paquete especial  `
And add:
`..así como los que requieren instalaciones especiales de paquetes`

Comment (ceoaliongroo) [inline README-es.md]: Nice done with "centrar" and "merecer"

File: README-es.md (hunk 6 above)
Comment (ceoaliongroo) [inline README-es.md]: :+1:

File: README-es.md (hunk 7 above)
Comment (ceoaliongroo) [inline README-es.md]: It's not necessary, please remove the changes.

File: README-es.md (hunk 8 above)
Comment (ceoaliongroo) [inline README-es.md]: It's not necessary, please remove the changes.

**Comment by ceoaliongroo** [comment]
@alu0100601398 I add more comment, please review it.

Thanks

File: README-es.md (hunk 3 above)
Comment (AndresCidoncha) [inline README-es.md]: I'm from Spain so I think that my perception is valid. "Experimentado" exist, but we don't use it.

Comment (ceoaliongroo) [inline README-es.md]: Ok, i understood you, but is used in a lot of countries in Latina America. So in this case we keep "Experimentado" 

Thanks for your opinion

File: README-es.md (hunk 8 above)
Comment (alu0100601398) [inline README-es.md]: Hi,but in the original version appears the word "do" -> Use Explainshell to get a helpful breakdown of what commands, options, pipes etc. do.
Can you check it?

File: README-es.md (hunk 7 above)
Comment (alu0100601398) [inline README-es.md]: Hi, I removed the changes, but it's really hard to read that sentence. I'll keep thinking about that.

File: README-es.md (hunk 5 above)
Comment (alu0100601398) [inline README-es.md]: Hi! Thank you so much
Back to the first version on the first change.
But in the second one:
"special package installs"
This refers to "special packages", isn't it?
I changed it to "..así como los que requieren instalaciones de paquetes especiales" .
And as they are in plural, I changed the next sentence to plural too.
"siempre que sean lo suficientemente importantes"

**Comment by alu0100601398** [comment]
Hi @ceoaliongroo I reviewed the changes. I let two comments for you.
Thank you for your time.

File: README-es.md (hunk 7 above)
Comment (ceoaliongroo) [inline README-es.md]: I did a double check and you are right

Please reprhase to:
 ...otros lugares buscando en Google una vez conoces la idea o el comando.

File: README-es.md (hunk 8 above)
Comment (ceoaliongroo) [inline README-es.md]: Good, 

Please change:
"las opcioneslas pipes," => "opciones, pipes"

**Comment by ceoaliongroo** [comment]
@aabreuglez good job, 

please after do the change merge with the master, to do the final check

thanks

File: README-es.md [hunk 9]
```diff
@@ -36,14 +36,14 @@ pero debido al interés mostrado, parecía valer la pena usar Github, donde pers
...
+- Se centra en Bash interactivo, aunque muchos de los consejos aplican para otros shells y al Bash scripting por lo general.
+- Incluye tanto comandos "estándar" Unix así como los que requieren instalaciones de paquetes especiales -- siempre que sean lo suficientemente importantes para merecer su inclusión.
 
 Notas:
 
-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
-- Usa [Explainshell](http://explainshell.com/) para obtener detalles de ayuda sobre que comandos, opciones, pipes, etc.
+-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
```
Comment (ceoaliongroo) [inline README-es.md]: Remove extra dash.

**Comment by jlevy** [comment]
Let me know if/how this gets resolved. Always good to keep the improvements coming in! Thanks for the efforts!

**Comment by ceoaliongroo** [comment]
@aabreuglez, do you need help to do the changes and merge with master?

**Comment by aabreuglez** [comment]
Sorry, @ceoaliongroo not much free time these days (exams season).
Merged the changes, updated. Fixed.
Check a note I let you
Also added you as a collaborator.

Regards

File: README-es.md (hunk 7 above)
Comment (aabreuglez) [inline README-es.md]: Done, much better now.

File: README-es.md (hunk 8 above)
Comment (aabreuglez) [inline README-es.md]: I  change the phrase to:
"Usa Explainshell para obtener detalles de ayuda sobre que hacen los comandos, las opciones, las pipes, etc."

Tried:
"sobre que hacen comandos, opciones, pipes, etc". But sounds a bit strange. What do you think?

**Comment by ceoaliongroo** [comment]
@aabreuglez Great!, thanks for take the time to do the changes. Good luck in the exams!

@jlevy, it's ready to merge, LGTM

**Comment by jlevy** [comment]
:+1: Thanks for the teamwork, @aabreuglez @ceoaliongroo !

**Pull request merged**